avm/res/sql/server/tests/e2e/db-identity/main.test.bicep

```diff
@@ -0,0 +1,79 @@
+targetScope = 'subscription'
```

Comment:
Hey @peterbud,
flawless work 💪 

I don't really have anything to add, but one suggestion: I'd suggest to integrate the DB-Identity test into the `max` test as this module is already deploying 11 SQL-Servers 😄

Reply:
Thanks, let me merge into the `max` test case

Reply:
Done